src/CMakeLists.txt

```diff
@@ -65,16 +65,17 @@ set(HEADER_FILES
   ${LIBCELLML_CONFIG_H}
 )
 
-set(CMAKE_CXX_VISIBILITY_PRESET hidden)
-set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
-
 add_library(cellml
   ${SOURCE_FILES}
   ${HEADER_FILES}
   ${MODULE_HEADER_FILES}
   ${API_HEADER_FILES}
 )
 
+set_target_properties(cellml PROPERTIES
+  CXX_VISIBILITY_PRESET hidden
+  VISIBILITY_INLINES_HIDDEN 1)
+
```

Comment:
Last thing and then I'm done, just a little further down in this file there is a set_target_properties command these two occurences should just be one.